addons/binding/org.openhab.binding.hive/README.md

```diff
@@ -0,0 +1,20 @@
+# Hive Binding
+
+Binding for British Gas's Hive Smarthome System
```

Comment:
The readme's purpose is also a bit of marketing for the binding and even for getting products of the manufacturer.
Can you please elaborate on "British Gas's Hive Smarthome System"? You can even add pictures.